controllers/utils/ramen.go

```diff
@@ -0,0 +1,251 @@
+package utils
```

Comment:
I've been thinking that we've been calling everything utils. Since this is part of the core functionality, its better we segregate into a new package for ramen related tasks.

Reply:
Can be done for other core functionalities as well for proper separation of concern.